src/components/Menu/Menu.stories.jsx

```diff
@@ -187,4 +187,111 @@ storiesOf('Components|Menu', module)
                 />
             </Menu>
         </div>
+    ))
+    .add('With Search', () => (
```

Comment:
When I have the addon panel the menus overlap each over. As it is the same behavior for all variants, I think we can have just one menu type to present the search behavior.

Reply:
Sounds good! I was mostly using it for testing when developing - I've removed the extra ones now and only left one.